drivers/pinmux/stm32/pinmux_stm32f0.h

```diff
@@ -31,4 +31,18 @@
 #define STM32F0_PINMUX_FUNC_PA15_USART2_RX \
 	(STM32_PINMUX_ALT_FUNC_1 | STM32_PUPDR_NO_PULL)
 
+#define STM32F0_PINMUX_FUNC_PB3_SPI1_SCK \
+	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
+#define STM32F0_PINMUX_FUNC_PB4_SPI1_MISO \
+	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
+#define STM32F0_PINMUX_FUNC_PB5_SPI1_MOSI \
+	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
+
+#define STM32F0_PINMUX_FUNC_PB13_SPI1_SCK \
```

Comment:
PB13/PB14/PB15 can be muxed to SPI2 only (not SPI1),

Reply:
Damn, it's a typo on both files...